### DISCUSSION TITLE
Traitor rework feedback

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#12309] [Unstable feedback]
The good:

- **(Most) Mission-related objectives are fun**. Preventing a beacon from blowing up was some of the most fun we've had.

- **XP/Money incentive is nice**. Gives a reason for individual players to accomplish their objectives at the detriment of the crew.

- **Does not promote griefing**. Most mission objectives don't revolve around killing your own crew or bricking the session, but instead trying to accomplish the objective without getting sussed out.

- **Customizeable difficulty/probability**

The bad:

- **Level 1 traitor missions:** they are most of the time too benign and unremarkable, there might not even be a traitor on board. It's hard to justify missions that really don't do much. If levels 1/2/3 are damage to equipment/missions/people, damage to equipment is usually shrugged off.

- **Objective variety depends on biome**. Since some of the later biomes (especially wastes) don't have outposts, the mission-related objectives do not spawn, and the ones that do are usually the same.

- **Captain (and security?) as traitors**. The goals of the traitor is antithetic to the goals of the captain. A traitor captain will either not want to sabotage his own ship, or will try to slow down the session to a crawl by poorly driving the sub around or giving conflicting orders. Security as a traitor is _too easy_, with complete access to the armory and just being strong in general. 

- **Voting system**. Simply put, a lot of players just forget to vote, and the voting UI is a little intrusive. There is no clear way of knowing if a majority of votes has been casted. Handcuffing the traitor just seems more straightforward and allows for the traitor to escape from the sub if detected and still achieve their objective

**Comment by ubertpendragon** [comment]
Specific mission feedback:

- **bakingstation**: probably the most fun mission, doesn't endanger the submarine, but a little easy to spot the traitor as that person will be the last to leave the station. Blowing up the reactor but getting voted as traitor fails the traitor mission and this is somewhat disappointing

- **cargotheft**: It's not very exciting. Most of the time nobody interacts with the cargo, besides people who steal and deconstruct said cargo, so it's relatively hard to detect. Can you just steal the items? Deconstruct them? Do you have to airlock them?

- **miningsabotage**: swapping oxygen for welding fuel on 80% is near impossible; it should check if a plasma cutter has exploded, and even that is relatively hard to accomplish if the divers have the plasma cutter in their inventory the whole time. Would benefit from _pickpocketing_

- **naturevsnurture**: skip the level 1 mission to the level 2 directly (hatching the egg). Most players will bring a mudraptor egg into the submarine whenever possible so level 1 is a non-objective

- **handcuffshortage**: like all traitor crafting missions, it's only as interesting as the item crafted; regular handcuffs aren't. Needs "different" handcuffs (faulty handcuffs that break after a few seconds?)

- **compoundnowhere**: This is the same as cargotheft, but level 1 somehow? It's about explosives it could be really more interesting/dangerous? Sabotage the explosives? Cause explosions in the following outpost level that kill NPCs? 

- **vipbombing**: really fun but really rare. VIP transports are among the scarcest missions in the game, and SPECIFICALLY a separatist VIP transport. It's also a level 2 mission when it should be level 3. 

- **deadoralive**: fun but really difficult, especially keeping sootman alive and not getting caught. The mask should prevent speaking, the ID should change team to hostile? (red)

- **undercover**: Good but would benefit from _pickpocketing_, and spawning a blank id card (removes the name but allows the victim to still open doors otherwise it's too noticeable)

- **armsinterception**: Same as compoundnowhere/cargotheft.

- **vippoison**: same rarity issue as vipbombing

- **showofstrength**: good one

- **deepdiverducks**: is the duck recipe exclusive to the traitor? Since ducks are now throwable, change the objective to causing concussions by throwing them at people?

- **theduffelbag**: enable deteriorating on the traitor duffel bag otherwise you can just dump it in the ballast with the others and no one will ever look inside. This forces you to move around and carry the bag from time to time. Add a sporadic tick sound coming from the bag to make it stand out? (not too often) 

- **theduffelbagbomb**: it's fun, 100 seconds to airlock the bomb might be a lot. Haven't experienced the defusal part?

- **theduffelbagbomb2**: bomb exploding on touching water is way too hazardous; a single leak and the ship explodes. Make it REALLY clear the bomb doesnt like water, or find another way for people not to throw the bomb (make it stick to hands like a pair of pseudo-handcuffs when you pick it up?)

- **harbingerofhonks**: like the handcuff mission, a bike horn is a very common item. Needs its own "fun" item (replacing gun ammo with a bike horn?)

- **divinelabor**: it works, noisy husk statues are annoying enough

- **faultyrobes**: very difficult objective, husk caller item is great

- **bloodsamples**:

- **bloodsamples2**:

**Comment by Mamuts123** [comment]
Nice
